src/func/entity-sets.ts

```diff
@@ -24,11 +24,13 @@ import {
 	createNewSetWithItems, getAddedItems, getComparisonFunc, getRemovedItems,
 	getUnchangedItems
 } from './set';
+import {type EntitySetMetadataT} from './entity';
+import type {ORM} from '..';
```

Comment:
Not really a good reason. I think my idea was that I'm importing the only type export from a module which otherwise only exports code. But that doesn't really make sense as we can easily change the import syntax if we were to also import code from this module in the future.
Applied as ee7e0625c37dd34789ec5a51a60deee4c009209c.
(Second attempt to submit this comment, my initial comment got lost when I applied the suggestion... bad UX.)